todo_app/templates/index.html

```diff
@@ -14,6 +14,7 @@ <h2>Items</h2>
         {% for item in items  %}
       
         <li> Id: {{item.id}} Title: {{ item.title}} Status: {{item.status}}</li>
+        {%if writeFlag%}
```

Comment:
You've hidden this functionality, but you haven't actually protected the endpoint so a user can bypass this (as mentioned above) but it does give them a better user experience once we've gotten past that. We should also hide the "create card" form so that people don't try to do that when they don't have writer permissions